tests/browser/heda_settings_accounts_and_contacts_edit_mode.robot

```diff
@@ -0,0 +1,92 @@
+*** Settings ***
+
+Resource        tests/HEDA.robot
+Suite Setup     Open Test Browser
+Suite Teardown  Delete Records and Close Browser
+
+*** Test Cases ***
+
+Go To Heda Home
+    Go To Heda Home
+    Select Frame			//iframe[contains(@id, "vfFrameId")]
+    Click Link				//a[contains(text(),'Accounts and Contacts')]
+    Wait Until Element Is visible	//a[contains(text(),'Accounts and Contacts')]
+    Capture Page Screenshot
+
+Validate Edit Mode For Accounts and Contacts, Settings
+    
+    
+    # Go To Heda Home
+    # Select Frame                    //iframe[contains(@id, "vfFrameId")]
+    # Click Element                   //div[@data-aura-rendered-by='106:0']//input[@type='radio']
+    # Capture Page Screenshot
+
+    #Go into Edit mode 
+    Click Element                   //div[@class='slds-button-group']//span[contains(text(), 'Edit')]
+    Wait Until Element Is visible   //div[@data-aura-rendered-by='28:0']
+    Capture Page Screenshot
```

Comment:
Remove use of Capture Page Screenshot.  When run on MetaCI, each screenshot is stored on Amazon S3.  When we start running this suite on all commits that eats up a lot of data storage.